lib/linters/example-linter.js

```diff
@@ -77,7 +77,7 @@ module.exports = async function (argv, tap) {
   }
 
   function extractPluginConfigs (exampleYaml) {
-    const pluginPattern = `${id}#v.+`
+    const pluginPattern = `${id}(#v.+)?`
```

Comment:
This makes version checking optional, which isn't preferable for public plugins, but I think it's a good change to make here. I wonder why we didn't get a failing test for this case though (public plugin without a version number)… could we add a test for that?

Let's keep this change here, but could we add some changes to lib/linters/readme-version-number-linter.js to ensure that:

1. We skip version number validation for local plugins (i.e. plugin id starts with "./")
1. We still validate version numbers for non-local plugins

Reply:
Ack. The alternative would be to make this `#(v.+|HEAD)` assuming that local plugins will always be referenced via `./path/to/plugin#HEAD`, which would be semantically what is happening. Theoretically I suppose local plugins don't always need to start with `./`, they could also start with `/` or `~/` or any other path magic?

Reply:
57488dd

Reply:
Thanks! I think that's probably enough.